java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/protocol/TestSuiteInfo.java

```diff
@@ -108,23 +109,23 @@ public void setFile(final String file) {
     }
 
     /**
-     * The line within the specified file where the suite definition starts (if known).
+     * The range within the specified file where the suite definition is located (if known).
      */
     @Pure
-    public Integer getLine() {
-        return line;
+    public Range getRange() {
```

Comment:
Is this class a base for a JSON message transferred between LSP backend and LSP client? If so, than this classifies as an API change. How are we going to document/version this kind of API?

Reply:
The API has not been marked stable or official yet. It is an extension to the LSP used by our VSCode plugin (sources of the plugin are part of the same module) as a client and it is currently work under development. Agree, the API should be documented once finalized.